Mathlib/Order/KrullDimension.lean

```diff
@@ -46,3 +53,81 @@ noncomputable def height (a : α) : WithBot (WithTop ℕ) := krullDim (Set.Iic a
 Coheight of an element `a` of a pre-ordered set `α` is the Krull dimension of the subset `[a, +∞)`
 -/
 noncomputable def coheight (a : α) : WithBot (WithTop ℕ) := krullDim (Set.Ici a)
+
+end definitions
+
+namespace krullDim
```

Comment:
I would rather not use a namespace here. `krullDim` is not a structure.

Reply:
this is done, thanks!